--- netinstall.yaml	2024-03-06 23:03:35.768640151 +0100
+++ netinstall.yaml.patched	2024-03-06 23:02:48.718201682 +0100
@@ -180,6 +180,14 @@
   packages:
     - firefox
     - firefox-i18n-$LOCALE
+- name: "Spell Checker and language package"
+  description: "Add spell check and language pack if possible"
+  hidden: false
+  selected: true
+  critical: false
+  packages:
+    - aspell
+    - aspell-$LOCALE
 - name: Firewall
   description: "Firewall installed and enabled"
   selected: true
