--- packagechooser.conf	2024-12-07 20:35:49.908555932 +0100
+++ packagechooser.conf.devel	2024-12-08 14:33:22.986879918 +0100
@@ -102,7 +102,7 @@
               description: "Unselect to install vanilla."
               selected: true
               packages:
-                - eos-plasma-sddm-config
+                - eos-breeze-sddm
                 - eos-settings-plasma
         packages:
             - ark
@@ -196,7 +196,7 @@
               selected: true
               packages:
                 - arc-gtk-theme-eos
-                - eos-settings-gnome
+                - eos-settings-gnome-devel
                 - eos-qogir-icons
         packages:
             - adwaita-icon-theme
@@ -280,7 +280,7 @@
                 - arc-gtk-theme-eos
                 - eos-lightdm-slick-theme
                 - eos-qogir-icons
-                - eos-settings-xfce4
+                - eos-settings-xfce4-devel
                 - endeavouros-xfce4-terminal-colors
         packages:
             - blueman
@@ -363,7 +363,7 @@
                 - arc-gtk-theme-eos
                 - eos-lightdm-slick-theme
                 - eos-qogir-icons
-                - eos-settings-cinnamon
+                - eos-settings-cinnamon-devel
         packages:
             - cinnamon
             - cinnamon-translations
@@ -433,7 +433,7 @@
             selected: true
             packages:
                 - arc-gtk-theme-eos
-                - eos-settings-mate
+                - eos-settings-mate-devel
                 - eos-lightdm-slick-theme
                 - eos-qogir-icons
         packages:
@@ -495,7 +495,7 @@
               selected: true
               packages:
                 - arc-gtk-theme-eos
-                - eos-settings-budgie
+                - eos-settings-budgie-devel
                 - eos-lightdm-slick-theme
                 - eos-qogir-icons
         packages:
