--- PKGBUILD	2024-02-19 14:14:00.739734635 +0100
+++ PKGBUILD.patched	2024-02-19 14:14:23.813391467 +0100
@@ -17,7 +17,7 @@
 'gptfdisk' 'hwinfo' 'kparts' 'polkit-qt6' 'python' 'solid' 'qt6-tools' 'libpwquality' 'ckbcomp' 'qt6-declarative' )
 provides=("calamares")
 options=(!strip !emptydirs)
-source=("git+https://github.com/endeavouros-team/$_reponame.git#branch=calamares")
+source=("git+https://github.com/killajoe/$_reponame.git#branch=calamares")
 
 sha256sums=('SKIP')
