--- prepare.sh	2025-06-25 13:45:22.016354070 +0200
+++ prepare.sh.patched	2025-06-27 13:03:43.947948957 +0200
@@ -1,5 +1,16 @@
 #!/usr/bin/env bash
 
+# add date to wallpaper
+mv airootfs/root/livewall.png airootfs/root/livewall-original.png
+
+magick airootfs/root/livewall-original.png \
+  -gravity NorthEast \
+  -pointsize 24 \
+  -fill white \
+  -font "DejaVu-Sans" \
+  -annotate +10+10 "$(date '+%Y-%m-%d %H:%M:%S')" \
+  airootfs/root/livewall.png
+
 # Get mirrorlist for offline installs
 wget -qN --show-progress -P "airootfs/etc/pacman.d/" "https://raw.githubusercontent.com/endeavouros-team/EndeavourOS-ISO/main/mirrorlist"
 
