--- reset.sh	2025-01-29 20:55:44.639558161 +0100
+++ reset.sh.patched	2025-02-13 19:09:17.177001785 +0100
@@ -1,9 +1,7 @@
 #!/bin/sh
-rm -rf "work" "out"
+rm -rf "work"
 rm airootfs/root/packages/*.pkg.tar.zst
-rm airootfs/root/packages/*.pkg.tar.zst.sig
 rm -rf airootfs/root/endeavouros-skel-liveuser/pkg
 rm airootfs/root/endeavouros-wallpaper.png
 rm airootfs/root/endeavouros-skel-liveuser/*.pkg.tar.zst
 rm -rf airootfs/etc/pacman.d/
-rm eosiso*.log
