--- run_before_squashfs.sh	2024-02-07 22:28:52.123405807 +0100
+++ run_before_squashfs.sh.patched	2024-02-07 22:32:01.202160449 +0100
@@ -72,6 +72,7 @@
 #systemctl enable NetworkManager.service systemd-timesyncd.service bluetooth.service firewalld.service
 #systemctl enable vboxservice.service vmtoolsd.service vmware-vmblock-fuse.service
 #systemctl enable intel.service
+systemctl start systemd-zram-setup@zram0.service
 systemctl set-default multi-user.target
 
 # Set wallpaper for live-session and original for installed system
