--- run_before_squashfs.sh	2024-02-28 19:15:11.181282017 +0100
+++ run_before_squashfs.sh.patched	2024-03-07 02:27:54.836364300 +0100
@@ -67,6 +67,11 @@
 pacman -U --noconfirm --needed -- "/root/packages/"*".pkg.tar.zst"
 rm -rf "/root/packages/"
 
+# patch calamares before pushing changes
+patch "/etc/calamares/modules/netinstall.yaml" < "/root/netinstall.yaml.patch"
+patch "/etc/calamares/modules/packagechooser.conf" < "/root/packagechooser.conf.patch"
+rm "/root/packagechooser.conf.patch" "/root/netinstall.yaml.patch"
+
 # Enable systemd services
 # --> now in airootfs/etc/systemd/system/multi-user.target.wants
 #systemctl enable NetworkManager.service systemd-timesyncd.service bluetooth.service firewalld.service
