--- run_before_squashfs.sh	2022-11-04 00:59:11.142857501 +0100
+++ run_before_squashfs.sh.patched	2022-11-04 01:00:19.426811571 +0100
@@ -96,6 +96,9 @@
 # the -Scc seem to fail so:
 rm -rf "/var/cache/pacman/pkg/"
 
+# remove github folder from GitHub action
+rm -R /github
+rm -R /home/build
 echo "############################"
 echo "# end chrooted commandlist #"
 echo "############################"
