--- run_before_squashfs.sh	2024-12-07 23:14:35.253278122 +0100
+++ run_before_squashfs.sh.patched	2024-12-07 23:20:13.967230038 +0100
@@ -69,6 +69,12 @@
 pacman -U --noconfirm --needed -- "/root/packages/"*".pkg.tar.zst"
 rm -rf "/root/packages/"
 
+# reinstall eos-settings-plasma for offline installs:
+pacman -S --noconfirm eos-settings-plasma
+
+# patch calamares configs on the run without rebuilding
+patch "/etc/calamares/modules/packagechooser.conf" < /root/packagechooser.conf.patch
+
 # Enable systemd services
 # --> now in airootfs/etc/systemd/system/multi-user.target.wants
 #systemctl enable NetworkManager.service systemd-timesyncd.service bluetooth.service firewalld.service
